chore(author-guide.md): elaborate on using MD and fix typos (#561)
todogroup/ospology: ospo-book/author-guide.md

> # Author's Guide
> Guidance on how to write content for the OSPO Book and prepare it for submission.

Reviewer: ```suggestion
# Author's Guide

Guidance on how to write content for the OSPO Book and prepare it for submission.
```

Same here

Author: Good catch

## Follow the chapter structure

## Use the writing style guide

## Use markdown

Markdown helps in writing content in a formatted way while making it easier to denote lists, headings, etc, without learning complex syntax. A beginner's guide for using basic markdown can be found [here](https://daringfireball.net/projects/markdown/basics). Markdown Syntax can be found [here](https://daringfireball.net/projects/markdown/syntax). When uploading to GitHub, the format needs to **stay** the same, i.e. it only needs to be in Markdown.

## Run pre-commit checks locally

> * Build the site locally to check if it looks right. 

Reviewer: Whitespace around headers is missing.

```suggestion

* Build the site locally to check if it looks right. 
```

Author: :+1:

# Author's Guide

Guidance on how to write content for the OSPO Book and prepare it for submission.

## Follow the chapter structure

## Use the writing style guide

## Use markdown

> Markdown helps in writing content in a formatted way while making it easier to denote lists, headings, etc, without learning complex syntax. A beginner's guide for using basic markdown can be found [here](https://daringfireball.net/projects/markdown/basics). Markdown Syntax can be found [here](https://daringfireball.net/projects/markdown/syntax). When uploading to GitHub, the format needs to **stay** the same, i.e. it only needs to be in Markdown.

Reviewer 2: ```suggestion
Markdown helps you to write content in a formatted way while making it easier to create lists, headings, etc, without learning complex syntax. [A beginner's guide for using basic markdown](https://daringfireball.net/projects/markdown/basics). Learn more about [Markdown Syntax](https://daringfireball.net/projects/markdown/syntax). Book content must be in Markdown format at the point you add it to the `todogroup/ospology` repository on GitHub.
```
